deploy/helm/kubecf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -225,10 +225,12 @@
   value: true
 
 {{- if .Values.features.suse_buildpacks.enabled }}
+# For SUSE buildpacks take all patches
```

Comment:
The check really should be against `features.suse_default_stack.enabled`, but that is outside the scope of this PR, and besides there are many other locations with this issue.

Reply:
Leaving it to separate PR